install/etc/services.available/20-mariadb-backup/run

```diff
@@ -1,35 +1,20 @@
 #!/usr/bin/with-contenv bash
 
 source /assets/functions/00-container
+# shellcheck disable=SC2034
 PROCESS_NAME="mariadb-backup"
 
 check_container_initialized
 check_service_initialized init 10-mariadb
 
 if [ "$1" != "NOW" ]; then
-    if [ ! -f /tmp/state/10-mariadb ]; then
+    if [ ! -f /tmp/state/10-mariadb-init ]; then
```

Comment:
This is related to my comment above in terms of where liftoff is. We should only do a backup if the actual mysqld runtime is operating, not the initialization mode.

Reply:
Remove?

Reply:
The more I look at it, you could get rid of that if statement entirely, it's redundant - The only thing we'd want to keep is the `sleep 10` statement inside of the if statement starting at line 9. I think as the db-backup image evolved and thus got ported over into this image I left some of the legacy scheduling checks in there. If this doesn't make sense I can perform the edit after merging your PR.